code/camera/cameradisplay.cpp

```diff
@@ -117,6 +127,28 @@ void CameraDisplay::update_zoom(int value) {
     Q_EMIT zoom_changed(zoom_factor);
 }
 
+void CameraDisplay::gridSelectChanged(int weight_index) {
```

Comment:
if `weight_index` isn't used leave out the name like
```
void CameraDisplay::gridSelectChanged(int) {...}
```

also function name should be snake case